lib/app.rb

```diff
@@ -0,0 +1,70 @@
+require 'sinatra'
+require 'sidekiq'
+require File.expand_path('../gems.rb', __FILE__)
+require File.expand_path('../workers/dependency.rb', __FILE__)
+require 'yaml'
+require 'redis-objects'
+
+module RubyDepsHelpers
+  def fetch_all(repositories)
+    repositories.each do |gem|
+      DependencyWorker.perform_async(gem.identifier, gem.gemspec_location, gem.gemfile_location)
+    end
+  end
+end
+
+config = <<YAML
+gollum:
+  gollum: true
```

Comment:
I think each project should also have a unique token of some sort that can be used to rerun the job by hitting an API endpoint (e.g., `/gollum/gollum/reassess/128ecf542a35ac5270a87dc740918404`) or `/gollum/gollum/reassess` met `128ecf542a35ac5270a87dc740918404` in the body van de request.

Reply:
Yes, good idea! GitHub actions actually allows you to set such secrets, so then we could trigger the reassesment from hook.